### PR TITLE
Add create aggregatorIntegration endpoint

### PR DESCRIPTION
**Pull request by Craiting**

### CODE REVIEW

File: apps/institution-service/src/controllers/aggregatorIntegrationController.ts [hunk 1]
```diff
@@ -38,3 +42,30 @@ export const updateAggregatorIntegration = async (
...
+  } catch (error) {
+    if (error instanceof ValidationError) {
+      res.status(400).json({
+        error: "Invalid aggregatorIntegration Data",
+        message: error.errors[0]?.message,
+      });
+    } else if (error instanceof Error) {
+      res.status(400).json({ error: "Database Error", message: error.message });
+    } else {
+      res.status(500).json({ error: "Error", message: "something went wrong" });
```
Comment (tphalp) [inline apps/institution-service/src/controllers/aggregatorIntegrationController.ts:68]: This is a nitpick, and not that important, but I'd use sentence case in any error message, just for consistency. "Something went wrong"

**Review by tphalp — APPROVED** [review]

File: apps/institution-service/cypress/e2e/aggregatorIntegrations.cy.ts [hunk 2]
```diff
@@ -223,3 +229,124 @@ describe("PUT /aggregatorIntegrations/:id (AggregatorIntegration update)", () =>
     method: "PUT",
   });
 });
+
+describe.only("POST /aggregatorIntegrations (Create)", () => {
```
Comment (wesrisenmay-mx) [inline apps/institution-service/cypress/e2e/aggregatorIntegrations.cy.ts:233]: only?

File: apps/institution-service/openApi.json [hunk 3]
```diff
@@ -761,6 +761,118 @@
             }
           }
         }
+      },
+      "post": {
```
Comment (wesrisenmay-mx) [inline apps/institution-service/openApi.json:765]: It looks like this is attached to the wrong url. When I load it in the swagger editor it says it requires an id

image

File: apps/institution-service/src/controllers/aggregatorIntegrationController.ts (hunk 1 above)
Comment (wesrisenmay-mx) [reply apps/institution-service/src/controllers/aggregatorIntegrationController.ts:68]: Agreed

File: apps/institution-service/src/controllers/aggregatorIntegrationController.ts [hunk 4]
```diff
@@ -38,3 +42,30 @@ export const updateAggregatorIntegration = async (
     });
   }
 };
+
+export const createAggregatorIntegration = async (
```
Comment (wesrisenmay-mx) [inline apps/institution-service/src/controllers/aggregatorIntegrationController.ts:46]: This is missing tests

**Review by wesrisenmay-mx — APPROVED** [review]

**Pull request merged**